### PR TITLE
Fix reducer for permalink and search

### PR DESCRIPTION
**Pull request by enahum**
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-9374

### CODE REVIEW

File: src/actions/posts.js [hunk 1]
```diff
@@ -508,7 +508,7 @@ export function getPostThread(postId, skipAddToChannel = true) {
             {
                 type: PostTypes.GET_POST_THREAD_SUCCESS
             }
-        ]), getState);
+        ], 'BATCH_GET_POST_THREAD'), getState);
```
Comment (jespino) [inline src/actions/posts.js]: I don't know exactly why this change. Is simply for be more explicit about the action type generated, for redux-devtools and for code clarity, or is there any place where we use this information?

Comment (enahum) [reply src/actions/posts.js]: This is mainly to use while debugging as is easier to find the action with all the `BATCH` actions everywhere, it does not have any implications

Comment (jespino) [reply src/actions/posts.js]: Sounds good :)

Comment (saturninoabril) [reply src/actions/posts.js]: @enahum Thanks for the info. I also learned from this :)
